app/models/form/sales/questions/staircase.rb

```diff
@@ -2,9 +2,9 @@ class Form::Sales::Questions::Staircase < ::Form::Question
   def initialize(id, hsh, page)
     super
     @id = "staircase"
-    @copy_key = "sales.sale_information.staircasing"
+    @copy_key = form.start_year_2025_or_later? ? "sales.setup.staircasing" : "sales.sale_information.staircasing"
```

Comment:
Can this just depend on the subsection id (as it would if it was using the default format) rather than the year? i.e. `"sales.#{subsection.id}.staircasing"`?